tests/integration/README.md

```diff
@@ -0,0 +1,32 @@
+# Integration scripts for Rancher
+
+
+## Development
+
+#### Install
+
+Setup virtualenv and enter, then: `pip install -r requirements.txt`
+
+#### Run tests
+
+Start a local rancher instance on port 8443
+
+Now you can execute the following to run:
+
+* the entire suite: `tox` (from tests/integration)
```

Comment:
This won't work unless someone actually installs tox. It isn't in requirements.txt.